Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "quicssh-rs"
-version = "0.1.4+autopublish"
+version = "0.1.5-dev"
```

Comment:
wired change, I think we make sure we change the version number when we do release.

Reply:
[Semantic Versioning](https://semver.org) defines version with `-` as pre-release version, and it is widely used to represent development version. Also, [The Cargo Book](https://doc.rust-lang.org/cargo/reference/resolver.html#pre-releases) says `cargo` supports this feature.
The latest release version is `0.1.4`, so in development we can/should use the pre-release versions of the next release version, such as `v0.1.5-alpha` or `v0.1.5-beta.2`. I chose `dev` for explicitness.